lexxpluss_apps/src/can_controller.cpp

```diff
@@ -149,7 +149,7 @@ class can_controller_impl {
     board_controller::msg_rcv_pb msg_board_to_pb{0};
     uint32_t prev_cycle_ros{0}, prev_cycle_send{0};
     bool heartbeat_timeout{false}, enable_lockdown{true};
-    static constexpr char version[]{"2.0.0"};
+    static constexpr char version[]{"3.0.1"};
```

Comment:
Since v3.0.0 has been released in the past, v3.0.1 seems to be correct.
v3.0.0は過去にリリースしているので、v3.0.1で正しそう。
https://github.com/LexxPluss/LexxHard-SensorControlBoard-Firmware/releases/tag/v3.0.0